e2e/dashboard-new-layouts/dashboards-panel-layouts.spec.ts

```diff
@@ -0,0 +1,324 @@
+import { e2e } from '../utils';
+
+describe('Dashboard', () => {
+  beforeEach(() => {
+    e2e.flows.login(Cypress.env('USERNAME'), Cypress.env('PASSWORD'));
+  });
+
+  it('can switch to auto grid layout', () => {
+    e2e.flows.scenes.importV2Dashboard({ title: 'Switch to auto grid' });
+
+    e2e.components.NavToolbar.editDashboard.editButton().click();
+
+    e2e.components.Panels.Panel.title('New panel').should('have.length', 3);
+
+    e2e.components.OptionsGroup.toggle('grid-layout-category').click();
+
+    e2e.flows.scenes.selectAutoGridLayout();
+
+    e2e.components.Panels.Panel.title('New panel').should('have.length', 3);
+
+    e2e.components.PanelEditor.ElementEditPane.AutoGridLayout.minColumnWidth().should('be.visible');
+    e2e.components.PanelEditor.ElementEditPane.AutoGridLayout.maxColumns().should('be.visible');
+    e2e.components.PanelEditor.ElementEditPane.AutoGridLayout.rowHeight().should('be.visible');
+    e2e.components.PanelEditor.ElementEditPane.AutoGridLayout.fillScreen().should('exist');
```

Comment:
Could factor this out since it's repeated below, just a small arrow function or something even